features/include/pcl/features/impl/moment_of_inertia_estimation.hpp

```diff
@@ -632,8 +541,7 @@ pcl::MomentOfInertiaEstimation<PointT>::setIndices (std::size_t row_start, std::
     return;
   }
 
-  indices_.reset (new std::vector<int>);
-  indices_->reserve (nb_cols * nb_rows);
+  indices_.reset (new std::vector<int>(nb_rows * nb_cols));
```

Comment:
Here you are creating a vector of `nb_rows * nb_cols` time `0`, doing the equivalent of `resize` instead of the original `reset`, is this normal?

Reply:
`indices_ = shared_ptr<std::vector<int>>`, so this shouldn't be different, right? I didn't quite get your `times 0` portion of the comment.

Reply:
`reserve` only changes the capacity of the vector, calling the constructor with an integer changes the size by creating `n` (here n = nb_rows * nb_cols) default-constructed elements (with the value `0` for `int`). See: http://coliru.stacked-crooked.com/a/821e1a25ebdbdd01

Reply:
Ahh, now I get it. n time 0 => n copies of 0